modules/ROOT/pages/6.6-release-notes.adoc

```diff
@@ -0,0 +1,256 @@
+New Premium plugin
+
+The following new Premium plugin was released alongside {productname} 6.6.
+
+=== AI Assistant 1.0.0
+
+The new Premium plugin, **AI Assistant**, allows a {productname} document to send queries to registered AI APIs and either:
+
+. insert it into the editor at the current selection;
+. create another query to further refine the response generated by the AI; or
+. close the dialog and discard the returned response.
+
+For information on **AI Assistant**, see xref:ai.adoc[AI Assistant].
+
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.6 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following updates:
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.6 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+
+[[additions]]
+== Additions
+
+{productname} 6.6 also includes the following additions:
+
+=== Added a new property value — `bottom` — for inline dialog configurations that anchors the dialog to the bottom of the editor
+//#TINY-9888
+
+A new property value — `+bottom+` — is available for inline dialog configuration.
+
+By default, {productname} dialogs present as modal and in the center of the editor viewport.
+
+Previously, adding a second argument — `{ inline: 'toolbar' }` to `editor.windowManager.open`, set the dialog to appear adjacent to the {productname} toolbar.
+
+With this update, a new value can be passed with this second argument: `{ inline: 'bottom' }`.
+
+This new inline dialog option sets the inline dialog to appear at the bottom of the editor viewport.
+
+For details, see the xref:dialog-configuration.adoc#dialog-position[Dialog position section] of the Dialog configuration documentation.
+
+=== Added a new property — `persistent` — for inline dialog configurations that will stop the dialog closing when clicking away from it
+//#TINY-9991
+
+A new, and optional, property, — `+persistent+` — is available for inline dialog configuration.
+
+When this property is set to `true`, an inline dialog will stay open when focus is switched away from the dialog and towards other parts of the {productname} editor (for example, the document editor, or a menu).
+
+If the property is not explicitly set, it is set to `+false+`, matching inline dialog behavior prior to this property being added to {productname}.
+
+Also, setting the property to `+true+` does not over-ride normal mechanisms for closing a dialog, such as clicking the _Close_ button or pressing the *Esc* key.
+
+For details, see the xref:dialog-configuration.adoc#configuration-parameters[Configuration parameters section] of the Dialog configuration documentation.
```

Comment:
There's a note section in the documentation part of the ticket. Should we include that as well? 

Since having this option, the inline dialog will not be closed when clicking away. When trying to open the same dialog when there's already one in the DOM with this option will lead to multiple instances of the same dialog created and shown in the editor. I think it would be great to add a note to recommend that the integrator is advised to have some checking to make sure that, when the dialog is already opened/exist in the DOM, they could use the `focus` method in `DialogInstanceApi` to transfer the focus into the dialog when it exists, instead of opening a new instance of the same dialog.

Reply:
@shanmen-tiny,

I’ve added a 2nd example to `dialog_persistent.adoc`: [_using `focus` to refocus an open but de-focused dialog_](https://github.com/tinymce/tinymce-docs/pull/2816/files#diff-9c9060a7eb7c0b2cf44fd1b539feb7d5d0e33428253ba8207d9f04b227e0c14c).

It needs attention from someone who know how to properly add the `focus` method to the example. :-)